src/kernel/arch.zig

```diff
@@ -1,2 +1,74 @@
+// Zig version: 0.4.0
+
+// ------------------------------------------------------------------------------------------------
+// Imports
+// ------------------------------------------------------------------------------------------------
+
+// ------------------------------------------------------------------------------------------------
+// Constants
+// ------------------------------------------------------------------------------------------------
+
+// ------------------------------------------------------------------------------------------------
+// Globals
+// ------------------------------------------------------------------------------------------------
+
+// ------------------------------------------------------------------------------------------------
+// Internal Functions
+// ------------------------------------------------------------------------------------------------
+
+// ------------------------------------------------------------------------------------------------
+// Public Functions
+// ------------------------------------------------------------------------------------------------
```

Comment:
These comments are unnecessary IMO, since it's quite clear that something is a constant or import without them. There are others elsewhere too.

Reply:
I thought i looked nioce for the separation, but can remove